ChangeLog

```diff
@@ -1,8 +1,23 @@
+<<<<<<< HEAD
```

Comment:
Please remove this conflict.

Reply:
This is a lesson to me not to generate the changelog too early. ;) LOL... this has been a PITA with every commit to master.

Reply:
fixed.